index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="navbar">
+        <h3 class="logo">Eg-Link</h3>
```

Comment:
- [ ] I think it would be great if you wrap your logo with `a` tag to make your code looks more professional. and because in the future you can link it to the first section so if anyone want to naigate can press on the logo and move easily 💯✅🔥🔥